src/implementations/twig/components/link/link.story.js

```diff
@@ -122,6 +123,19 @@ const getArgTypes = () => ({
       category: 'Icon',
     },
   },
+  icon_title: {
+    name: 'icon title',
+    type: 'string',
+    description: 'Textual information for the icon, mostly for screen readers',
+    if: { arg: 'external', truthy: false },
```

Comment:
mmh..we just investigated the possibility of defining multiple conditions and unfortunately that is not currently possible. With this syntax basically only the last condition is considered, so to achieve this we should kind of duplicate the control splitting the conditions. But it seems unneeded here, we are not able to build the perfect story, but the issues we have are minor,  when an icon is selected and then the external control is activated we do have unneeded controls for the icon (transform, but probably also position), when the external is set to true and no icon was previously chosen then no additional control (so even the icon title) will be visible.